docs-src/content/getting-started/streaming.md

```diff
@@ -69,4 +69,37 @@ func main() {
 }
 ```
 {{% /tab %}}
+
+{{% tab "Python" %}}
+``` python
+import cubic
+
+serverAddress = '127.0.0.1:2727'
+
+client = cubic.Client(serverAddress)
+
+# use the first available model
+modelResp = client.ListModels()
+model = modelResp.models[0]
+
+cfg = cubic.RecognitionConfig(
```

Comment:
Some extra config options in here might be helpful to show (I assume this is now where things like asking for the CN, turning on formatting etc are configured). 

In general, I'd suggest showing more (or better, all) options in the sample code assuming customers can remove them if they don't want them. But a customer working out how to add a new option will take them some effort.

Reply:
Good idea. We could add a section in the docs (across languages) to show how to configure a few useful things, such as these.

Reply:
will do this in a separate PR, resolving for now